angular/src/app/modules/booking/booking-routing.module.ts

```diff
@@ -4,7 +4,9 @@ import { RouterModule, Routes } from '@angular/router';
 import { BookingComponent } from './booking/booking.component';
 import { SearchResultsComponent } from './search-results/search-results.component';
 
-const routes: Routes = [{ component: BookingComponent, path: '' }];
+const routes: Routes = [
+  { component: BookingComponent, path: '' }];
+
```

Comment:
Are you using Prettier? Is this a result of that or the default VSCode formatter?